server/token.go

```diff
@@ -35,11 +37,14 @@ const (
 
 // generate a token
 func token(length int) string {
-	result := ""
+	result := make([]byte, length)
 	for i := 0; i < length; i++ {
-		x := rand.Intn(len(SYMBOLS) - 1)
-		result = string(SYMBOLS[x]) + result
+		x, err := rand.Int(rand.Reader, big.NewInt(int64(len(SYMBOLS))))
+		if err != nil {
```

Comment:
```suggestion
		if err != nil {
		// log an error entry and fallback to `math/rand`
```

Reply:
I agree, will do it later. Should I pass logger into function?

Reply:
yes, thanks

you should pass a null logger in the test: https://github.com/dutchcoders/transfer.sh/blob/main/server/token_test.go

Reply:
I did it. But error message in logger in this function provide three error messages in log, because token calls several times.